invenio_communities/config.py

```diff
@@ -21,8 +21,10 @@
 
 COMMUNITIES_ROUTES = {
     "frontpage": "/communities",
-    "search": "/communities/search",
-    "new": "/communities/new",
+    "search": "/communities-search",
+    "deprecated_search": "/communities/search",
+    "new": "/communities-new",
+    "deprecated_new": "/communities/new",
```

Comment:
minor: I think just adding them doesn't automatically register the routes as well, they have to be registered inside the `invenio_communities/views/ui::create_ui_blueprint` function.